data/Identifying_production_data_types.md

```diff
@@ -1,3 +1,216 @@
+* RMA_normalized : An integer if 1 means data was RMA normalized.
+
+###ExpressionPlatform
+####Description
+The platform that the experiment was run on. Generally for microarray, but potentially some instrument information for other technology types could be stored in this.  This generally maps to a GEO GPL.
+####Relationships
+It references Genomes.
+Referenced by ExpressionSamples
+####Fields:
+* id : Id of the series usually gotten from the id-server for public data
+* source_id : Some string representing the source, in the case of GEO it would be the GPL ID
+* genome_id : WS reference to a genome object.
+* strain : A strain subtype (genome_id, reference_strain, wild_type, description, name)
+* technology : The technology used
+* title : If the platform/chip design was testing for a certain thing.
+
+###ExpressionReplicateGroup
+####Description
+A simple object that stores ws references to Expression samples that are considered to be replicates of one another.
+####Relationships
+It references ExpressionSamples
+####Fields:
+* id : Id of the series usually gotten from the id-server for public data
+* expression_sample_ids : A list of WS References to ExpressionSamples that are replicates of one another.
+
+###RNASeqSample
+####Description
+Made by Sri.  An RNA-Seq Sample.
+####Relationships
+None of the references are WS references. It does have ref_genome, po_id, eo_id.
+It does have a shock_url and shock_id, but these are only set up as strings.
+####Fields:
+* name  
+* type
+* created
+* shock_ref (shock_id and shock_url)A shock url to the read files for RNA-Seq experiments
+* metadata
+  * paired
+  * platform
+  * sample_id
+  * title
+  * source
+  * ext_source_date
+  * domain
+  * ref_genome
+  * tissue - a list
+  * condition - a list
+  * po_id - a list of PO terms
+  * eo_id - a list of EO terms
+
+###RNASeqSampleAlignment
+####Description
+Made by Sri.  Alignment of the RNA-Seq Sample? Nearly identical to RNASeqSample.
+####Relationships
+None of the references are WS references. It does have ref_genome, po_id, eo_id.
+It does have a shock_url and shock_id, but these are only set up as strings.
+####Fields:
+* name  
+* paired
+* created
+* shock_ref (shock_id and shock_url) A shock url to the read files for RNA-Seq experiments
+* metadata
+  * paired
+  * platform
+  * sample_id
+  * title
+  * source
+  * ext_source_date
+  * domain
+  * ref_genome
+  * tissue - a list
+  * condition - a list
+  * po_id - a list of PO terms
+  * eo_id - a list of EO terms
+
+###RNASeqDifferentialExpression
+####Description
+Made by Sri.  Differential Expression
+####Relationships
+It does have a shock_url and shock_id, but these are only set up as strings.
+####Fields:
+* name  
+* title
+* created
+* diff_expression
+  *  RNASeqDifferentialExpressionSet - list of RNASeqDifferentialExpressionFiles (contains name, shock_ref(shock_url,shock_id) A shock url to the read files for RNA-Seq experiments
+
+---
+
+#Assembly
```

Comment:
Sorry confused about this entry, do you reference KBaseAssembly from your type module?